templates/react/{{cookiecutter.project_slug}}/{{cookiecutter.module_name}}/frontend/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "my_localstack_extension",
+  "version": "0.1.0",
+  "private": true,
+  "license": "UNLICENSED",
+  "dependencies": {
+    "@emotion/react": "^11.11.4",
+    "@emotion/styled": "^11.11.5",
+    "@localstack/theme": "git+ssh://git@github.com/localstack/localstack-web.git#commit=741276a31896cb498dc6207830842e6cbcca9611&workspace=@localstack/theme",
+    "@mui/material": "^5.15.20",
+    "@testing-library/react": "^13.4.0",
+    "@types/node": "^16.18.99",
+    "@types/react-dom": "^17.0.11",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-router-dom": "^6.24.0",
+    "react-scripts": "5.0.1",
+    "typescript": "^4.9.5"
+  },
+  "devDependencies": {
+    "@esbuild-plugins/node-modules-polyfill": "^0.1.4",
+    "concurrently": "^8.2.2",
+    "esbuild": "^0.16.6",
+    "esbuild-envfile-plugin": "^1.0.2",
+    "esbuild-plugin-clean": "^1.0.1",
+    "esbuild-plugin-copy": "^0.3.0",
+    "esbuild-plugin-svgr": "^1.0.0",
+    "live-server": "^1.2.2"
+  },
+  "scripts": {
+    "start": "concurrently --restart-tries -1 --raw \"yarn serve\" \"yarn watch\"",
+    "serve": "live-server --port=3000 --host=0.0.0.0 --no-browser --watch=build/index.html --entry-file=index.html build",
+    "watch": "node .esbuild --watch",
+    "build": "node .esbuild"
+  },
+  "resolutions": {
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "@types/react-dom": "^17.0.11",
+    "@mui/material": "^5.15.20"
+  },
+  "packageManager": "yarn@3.2.3"
```

Comment:
perhaps it would be nice for npm noobs like myself to add a line into the readme that this requires yarn3 and how to find installation instructions).

here's the error you get after `make install`:

```
error This project's package.json defines "packageManager": "yarn@3.2.3". However the current global version of Yarn is 1.22.22.

Presence of the "packageManager" field indicates that the project is meant to be used with Corepack, a tool included by default with all official Node.js distributions starting from 16.9 and 14.19.
Corepack must currently be enabled by running corepack enable in your terminal. For more information, check out https://yarnpkg.com/corepack.
make: *** [Makefile:27: install-frontend] Error 1
```